docs/gen_ref_pages.py

```diff
@@ -0,0 +1,56 @@
+"""Generate the code reference pages.
+
+Based on https://mkdocstrings.github.io/recipes/#generate-pages-on-the-fly
+"""
+
+import pathlib
+
+import mkdocs_gen_files
+
+nav = mkdocs_gen_files.Nav()
+
+for path in sorted(pathlib.Path("hikari").rglob("*.py")):
+    module_path = path.relative_to(".").with_suffix("")
```

Comment:
Probably shouldn't rely on CWD for this?

Reply:
mkdocs will ensure they get build in a specific way, which i would rather than traversing the paths